src/main/java/ch/njol/skript/command/CommandHelp.java

```diff
@@ -68,8 +74,27 @@ private CommandHelp(String command, String argsColor, String node) {
 		this.argsColor = argsColor;
 	}
 
+	/**
+	 * Add an argument to the CommandHelp.
+	 * 
+	 * @param argument String that represents the node path to the help info.
+	 * @return This CommandHelp for building.
+	 */
 	public CommandHelp add(String argument) {
+		return add(argument, null);
+	}
+
+	/**
+	 * Add an argument to the CommandHelp.
+	 * 
+	 * @param argument String that represents the node path to the help info.
+	 * @param pattern The pattern to match against user input.
+	 * @return This CommandHelp for building.
+	 */
+	public CommandHelp add(String argument, @Nullable String pattern) {
 		ArgumentHolder holder = new ArgumentHolder(argument);
+		if (pattern != null)
+			holder.pattern = PatternCompiler.compile(pattern);
```

Comment:
I don't see a useful reason for adding skript patterns to skript command arguments. A simple aliases param is enough.